### ISSUE TITLE
WMV files error

### ISSUE THREAD

**Issue by eagleeyez** [RandomEngy/VidCoder#155]
I've an issue with some wmv files that doesn't open... I did a quick fix by replacing the hb.dll with a version from handbrake nightly build and it fix the issue and I've created a portable version out of it but I guess it will be good idea to update it on this project so other users will not have to deal with the same issue.

**Comment by RandomEngy** [comment]
I released 2.41 Beta, which uses HandBrake 1.0.0 core. You could give it a try with that version.

**Comment by eagleeyez** [comment]
OkKI will give it a shot but I think that the dll from the 1.0.0 version don't fix the issue but the latest nightly builds did... I will get back within few minutes.

Thanks.

**Comment by eagleeyez** [comment]
The links on the download page are broken and I cannot access to the release folder on github

**Comment by eagleeyez** [comment]
I tried this and as expected it didn't work, also the portable version is broken and don't launch... The url on the homepage are not linking to the files as well
![image](https://cloud.githubusercontent.com/assets/1388336/21557954/35a053ce-ce01-11e6-8487-92e881aa596f.png)

**Comment by RandomEngy** [comment]
Thanks, the links from http://vidcoder.net/ should be fixed now. I was able to get at them from the GitHub page though: https://github.com/RandomEngy/VidCoder/releases/tag/v2.41-beta . I haven't had trouble running the portable version though. What happens exactly when you try to launch?

If this is happening on HandBrake 1.0.0 as well, you can report the problem to them and they can fix it.

**Comment by eagleeyez** [comment]
Yes good idea, I will report the issue with wmv to handbrake authors... (I know it's fixed now on nightly builds but maybe they have to fix it on the newly released version)

Thank You and I cannot wait to seem more features and improvements to this amazing software (I used tons of transcoder during several years and I think vidcoder is the most reliable so far and user friendly) maybe adding some x265 options will make it even better.

**Comment by RandomEngy** [comment]
I think it's a recent regression. The 1.0.0 build right now is brand new and more up to date than the nightlies I believe. They are working on 1.0.1 now, though. 

Glad you like it. x265 options is on the backlog.
